app/src/main/res/values-nl/strings.xml

```diff
@@ -1,66 +1,66 @@
+    <string name="show_on_map">Weergeven op kaart</string>
+    <string name="unknown_location">Onbekende locatie</string>
+    <string name="no_map_application">Geen kaarten-app gevonden</string>
+    <string name="no_camera_app_found">Geen camera-app gevonden</string>
     <string name="increase_column_count">Meer kolommen</string>
     <string name="reduce_column_count">Minder kolommen</string>
-    <string name="change_cover_image">Afbeelding voor omslag veranderen</string>
+    <string name="change_cover_image">Omslagafbeelding wijzigen</string>
     <string name="select_photo">Foto selecteren</string>
     <string name="use_default">Standaard gebruiken</string>
     <string name="volume">Volume</string>
     <string name="brightness">Helderheid</string>
-    <string name="do_not_ask_again">Onthouden voor deze sessie</string>
-    <string name="lock_orientation">Schermrotatie vergrendelen</string>
-    <string name="unlock_orientation">Schermrotatie ontgrendelen</string>
+    <string name="do_not_ask_again">Niet opnieuw vragen (in deze sessie)</string>
+    <string name="lock_orientation">Schermoriëntatie vergrendelen</string>
+    <string name="unlock_orientation">Schermoriëntatie ontgrendelen</string>
 
     <!-- Filter -->
-    <string name="filter_media">Filter media</string>
+    <string name="filter_media">Media filteren</string>
     <string name="images">Afbeeldingen</string>
     <string name="videos">Video\'s</string>
-    <string name="gifs">GIF-bestanden</string>
-    <string name="no_media_with_filters">Geen bestanden gevonden met de huidige filters.</string>
+    <string name="gifs">GIF\'s</string>
+    <string name="no_media_with_filters">Er zijn geen bestanden gevonden met de huidige filters.</string>
     <string name="change_filters_underlined"><u>Filters aanpassen</u></string>
 
     <!-- Hide / Exclude -->
-    <string name="hide_folder_description">Deze functie verbergt de map door het bestand \'.nomedia\' toe te voegen. Alle submappen zullen ook worden verborgen. Kies \'Verborgen mappen tonen\' in de instellingen om toch verborgen mappen te kunnen inzien. Doorgaan?</string>
+    <string name="hide_folder_description">Deze functie verbergt de map door de extensie \'.nomedia\' toe te voegen. Alle submappen zullen ook worden verborgen. Kies \'Verborgen mappen weergeven\' in de instellingen om verborgen mappen alsnog te weergeven. Wilt u doorgaan?</string>
     <string name="exclude">Uitsluiten</string>
     <string name="excluded_folders">Uitgesloten mappen</string>
     <string name="manage_excluded_folders">Uitgesloten mappen beheren</string>
     <string name="exclude_folder_description">De selectie en submappen uitsluiten van deze galerij. Uitgesloten mappen kunnen worden beheerd in de instellingen.</string>
-    <string name="exclude_folder_parent">Map die dit item bevat uitsluiten?</string>
+    <string name="exclude_folder_parent">Wilt u de map met dit item uitsluiten?</string>
```

Comment:
I avoided any "je"/"u" stuff where possible, plus it's more concise.

Reply:
That's a matter of opinion. I think 'u' is more appropriate, but oh well.